proto/ibc/core/client/v2/params.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package ibc.core.client.v2;
+
+option go_package = "github.com/cosmos/ibc-go/v10/modules/core/02-client/v2/types";
+
+// Params is a **per-client** parameter that sets which relayers are allowed to relay v2 IBC messages
+// for a given client.
+// If it is set, then only relayers in the allow list can send v2 messages
+// If it is not set, then the client allows permissionless relaying of v2 messages
+message Params {
```

Comment:
Maybe should be called `ClientParams`. As this can be confused with a cosmos-sdk module param